src/mesa/sale.ts

```diff
@@ -42,7 +42,7 @@ export const isSaleClosed = ({ endDate }: Sale) => {
  * @param order1
  * @param order2
  */
-export const isBidActive = ({ bids }: Sale, bid: SaleBid) => {
+export const isBidActive = ({ bids }: Sale, bid: FairSaleBid & FixedPriceSalePurchase) => {
```

Comment:
`FixedPriceSalePurchase` doesn't have a status. Every purchase goes through if it meets requirements: `min allocation >= price <= max allocation`